RetailCoder.VBE/Root/RubberduckModule.cs

```diff
@@ -306,6 +306,11 @@ private void BindQuickFixTypes(IEnumerable<Assembly> assemblies)
             foreach (var quickFix in quickFixes)
             {
                 Bind<IQuickFix>().To(quickFix).InSingletonScope();
+                if (quickFix.Name.Equals("AssignedByValParameterMakeLocalCopyQuickFix"))
```

Comment:
Yep. Deleting [AssignedByValParameterQuickFixDialogFactory.cs](https://github.com/rubberduck-vba/Rubberduck/blob/427ef6015b22275c178eafe10ff8d508e8c50c32/RetailCoder.VBE/UI/Refactorings/AssignedByValParameterQuickFixDialogFactory.cs) should fix this: Ninject will apply the `IFooFactory` binding convention and generate the trivial factory implementation for us.